amnestywebsite/humanity-theme
# Background Media PHP Register and Render

Ref: https://github.com/amnestywebsite/humanity-theme/issues/146

**Steps to test**:
1. Create/Edit a post
2. Check Background Media block appears in the block inserter
3. Insert the block and populate with content
4. Save and view the front end, it should display as expected

Example:
http://bigbite.im/i/uavRkE
